taui/src/actions/json-data.js

```diff
@@ -3,11 +3,11 @@ import fetch from '@conveyal/woonerf/fetch'
 
 import cacheURL from '../utils/cache-url'
 
-export function loadPointsOfInterest (url: string) {
+export function loadDataFromJSON (url: string, type: string) {
```

Comment:
This is an [action creator](https://redux.js.org/basics/actions#action-creators). The data reducer matches on the `type` label.